### PR TITLE
feat: Activity tab view on projects

### PR DESCRIPTION
**Pull request by ncomerci**
Part of https://github.com/decentraland/governance/issues/1776

image
image

### CODE REVIEW

File: src/components/Home/ActivityTicker.tsx [hunk 1]
```diff
@@ -70,24 +68,16 @@ export default function ActivityTicker() {
...
+  const { events, refetch, isLoading } = useEvents({ event_type: parseTickerFilter(filterState) })
 
   const handleApply = (filters: TickerFilter) => {
     setFilterState(filters)
   }
 
   useEffect(() => {
     refetch()
-  }, [filterState, refetch])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (andyesp) [inline src/components/Home/ActivityTicker.tsx:79]: Why did you remove refetch from the exhaustive deps?

File: src/components/Projects/ActivityTab.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          {events && events.length === 0 && (
+            <div className="ActivityTicker__EmptyContainer">
+              <Empty description={t('page.home.activity_ticker.no_activity')} />
+            </div>
+          )}
+          {events && events.length > 0 && (
+            <div className="ActivityTicker__List">
+              {events.map((item) => {
+                return (
+                  <div key={item.id} className="ActivityTicker__ListItem">
```
Comment (andyesp) [inline src/components/Projects/ActivityTab.tsx:34]: Maybe we can create a new `ActivityTicketListItem` component to reuse here and in `ActivityTicker.tsx`?

Comment (andyesp) [reply src/components/Projects/ActivityTab.tsx:34]: or `ActivityTicketList`

File: src/components/Home/ActivityTicker.tsx (hunk 1 above)
Comment (ncomerci) [reply src/components/Home/ActivityTicker.tsx:79]: it was causing me multiple re-rerenders

**Review by 1emu — APPROVED** [review]
